qa/workunits/rbd/rbd_mirror.sh

```diff
@@ -167,22 +168,14 @@ stop_mirror()
 	done
 	ps auxww | awk -v pid=${pid} '$2 == pid {print; exit 1}'
     fi
-    rm -f $(daemon_asok_file "${daemon}" "${CLUSTER1}")
-    rm -f $(daemon_asok_file "${daemon}" "${CLUSTER2}")
+    rm -f $(daemon_asok_file "${daemon}")
     rm -f $(daemon_pid_file "${daemon}")
 }
 
 daemon_local_cluster()
 {
     local daemon=$1
-    local pid_file=$(daemon_pid_file $daemon)
-
-    pid=$(cat $(daemon_pid_file "${daemon}"))
-
-    test -n "${pid}"
-
-    ps auxww | awk -v pid=${pid} '$2 == pid' |
-	sed -nEe 's/^.*--cluster +([^ ]+).*$/\1/p'
+    echo "${daemon}" | sed s/_daemon$//
```

Comment:
You could use:
```
echo ${daemon%_daemon}
```

But actually, although I like the idea to make automatic detection, I don't like very much that you relay on daemon name being "${CLUSTER}_daemon", while CLUSTER{X}_DAEMON vars are defined in the header section, so a user could change them for her purposes, e.g. set `CLUSTER1_DAEMON=primary` , and it will break the script.

Thus I would remove CLUSTERXX_DAEMON vars then, so they are built by the script internally and users don't have an impression that daemon name can be redefined. And in functions, instead of passing `daemon` arg, you could pass `cluster` arg and `daemon` var is built locally as ${cluster}_daemon.